MsmqReturnToSourceQueue/MsmqUtilities.cs

```diff
@@ -179,21 +181,26 @@ static string GetCorrelationId(Message message, Dictionary<string, string> heade
         return result;
     }
 
-    public static string GetFullPathWithoutPrefix(MsmqAddress address)
+    public static void SaveMessageHeaders(Dictionary<string, string> headers, Message m)
```

Comment:
For consistency, I would use the same mechanisms as in the [original source for this file](https://github.com/Particular/NServiceBus/blob/5.2.25/src/NServiceBus.Core/Transports/Msmq/MsmqUtilities.cs)

Reply:
done